sr_utilities_common/launch/bimanual_simple_kinematic_simulator.launch

```diff
@@ -0,0 +1,58 @@
+<launch>
+  <!-- Arm model: ur10 or ur10e -->
+  <arg name="robot_model" default="ur10e"/>
+  <arg name="robot_description" default="$(find sr_multi_description)/urdf/bimanual_srhand_$(arg robot_model).urdf.xacro"/>
+
+  <param name="robot_description" command="xacro --inorder $(arg robot_description)"/>
+
+  <!-- Right hand -->
+  <include file="$(find sr_utilities_common)/launch/single_robot_position_republisher.launch">
+    <arg name="node_name" value="rh_trajectory_republisher"/>
+    <arg name="remapped_joint_states_topic_name" value="rh_remapped_joint_states"/>
+    <arg name="trajectory_cmd_topic_name" value="/rh_trajectory_controller/command"/>
+    <arg name="joints_to_move" value="[]"/>s
```

Comment:
there is an 's' sign at the end of this line and a few others

Reply:
fixed